idaes/generic_models/unit_models/plate_heat_exchanger.py

```diff
@@ -291,34 +297,38 @@ def _make_params(self):
         _total_active_plate_number = 2 * sum(self.config.channel_list) - 1 -\
             self.config.divider_plate_number
 
-        self.plate_area = Expression(expr=self.config.total_area /
+        self.plate_area = Expression(expr=self.config.total_area*pyunits.m**2 /
                                      _total_active_plate_number,
                                      doc="Heat transfer area of single plate")
 
         # Plate gap
         if self.config.plate_gap is None:
             _total_plate_number = 2 * sum(self.config.channel_list) + 1 +\
                 self.config.divider_plate_number
-            _plate_pitch = self.config.plate_pact_length / _total_plate_number
+            _plate_pitch = self.config.plate_pact_length*pyunits.m / _total_plate_number
 
-            _plate_gap = _plate_pitch - self.config.plate_thickness
+            _plate_gap = _plate_pitch - self.config.plate_thickness*pyunits.m
         else:
-            _plate_gap = self.config.plate_gap
+            _plate_gap = self.config.plate_gap*pyunits.m
 
         self.plate_gap = Expression(expr=_plate_gap)
```

Comment:
As above - it seems like a really bad idea to build the config arguments directly into the Expressions.